test/test_datasets.py

```diff
@@ -149,51 +149,6 @@ def test_fashionmnist(self, mock_download_extract, mock_check_integrity):
             img, target = dataset[0]
             self.assertEqual(dataset.class_to_idx[dataset.classes[0]], target)
 
-    @unittest.skipIf(sys.platform in ('win32', 'cygwin'), 'temporarily disabled on Windows')
-    def test_cityscapes(self):
-        with cityscapes_root() as root:
-
-            for mode in ['coarse', 'fine']:
-
-                if mode == 'coarse':
-                    splits = ['train', 'train_extra', 'val']
-                else:
-                    splits = ['train', 'val', 'test']
-
-                for split in splits:
-                    for target_type in ['semantic', 'instance']:
-                        dataset = torchvision.datasets.Cityscapes(
-                            root, split=split, target_type=target_type, mode=mode)
-                        self.generic_segmentation_dataset_test(dataset, num_images=2)
-
-                    color_dataset = torchvision.datasets.Cityscapes(
-                        root, split=split, target_type='color', mode=mode)
-                    color_img, color_target = color_dataset[0]
-                    self.assertTrue(isinstance(color_img, PIL.Image.Image))
-                    self.assertTrue(np.array(color_target).shape[2] == 4)
-
-                    polygon_dataset = torchvision.datasets.Cityscapes(
-                        root, split=split, target_type='polygon', mode=mode)
-                    polygon_img, polygon_target = polygon_dataset[0]
-                    self.assertTrue(isinstance(polygon_img, PIL.Image.Image))
-                    self.assertTrue(isinstance(polygon_target, dict))
-                    self.assertTrue(isinstance(polygon_target['imgHeight'], int))
-                    self.assertTrue(isinstance(polygon_target['objects'], list))
-
-                    # Test multiple target types
-                    targets_combo = ['semantic', 'polygon', 'color']
-                    multiple_types_dataset = torchvision.datasets.Cityscapes(
-                        root, split=split, target_type=targets_combo, mode=mode)
```

Comment:
While we now check multiple `target_type`'s the old test also checks if the returned content is valid. I did something similar for `Caltech101`:

https://github.com/pytorch/vision/blob/19ad0bbc5e26504a501b9be3f0345381d6ba1efc/test/test_datasets.py#L415

Maybe we can do the same here. Thoughts?